### ISSUE TITLE
Keras model.evaluate does not have output.

### ISSUE THREAD

**Issue by AsakusaRinne** [SciSharp/TensorFlow.NET#986]
The `model.evaluate` in keras should output the metric of the model to terminal but not.
